app/lib/github.rb

```diff
@@ -15,6 +15,16 @@ def self.client
     )
   end
 
+  def self.get_file(repo, path)
+    client.contents(repo, path: path, headers: { 'Accept': 'application/vnd.github.v3.raw' })
+  rescue Octokit::NotFound
+    false
+  end
+
+  def self.check_file(repo, path)
+    !!get_file(repo, path)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid the use of double negation (`!!`).](https://app.codacy.com/manual/VoxPupuli/vox-pupuli-tasks/pullRequest?prid=5444834)